### DISCUSSION TITLE
Strider/Lavagantes renomeados

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#1053] [Ideia]
### Servidor afetado

Todos

### Qual o propósito da ideia?

Adicionar mais um mob como pet

### Descrição da ideia

Permitir que o mob em questão permaneça após a limpeza do chão ao colocar nametag.

### Aplicação

Apenas alterar as permissões na limpeza.

### Referências

Desejos de players.

**Comment by leogianfagna** [comment]
@Batatoones faça como um issue e coloque em tarefas que faço assim que dar

**Comment by leogianfagna** [comment]
deixa que eu coloco
